htdocs/web_portal/views/site/view_site.php

```diff
@@ -649,6 +650,32 @@ class="header"
                             }
                             ?>
                         </td>
+                        <td style="text-align:center">
+                            <?php
+                            $useTime = $APIAuthEnt->getLastRenewTime();
+                            $titleStr = 'Last renewed ' . $useTime->format('d-m-Y H:iTP');
+                            echo '<div title="' . $titleStr . '">';
+                            ?>
+                            <?php if (!$portalIsReadOnly) :?>
+                            <form action="index.php?Page_Type=Edit_API_Authentication_Entity&amp;
+                                          authentityid=<?php echo $APIAuthEnt->getId();?>&amp;
+                                          isRenewalRequest=true"
+                                   method="post">
+                                <button type="submit">
+                                <?php
+                                    echo '<img height="18px" src="'
+                                        . \GocContextPath::getPath()
+                                        . 'img/refresh_key.png"';
```

Comment:
I wonder if an icon is the best way to go here, or to have a seperate renew button similar to the "Edit" and "Delete" buttons - something to discuss at the hackathon :)

Reply:
From discussions in the hackathon, this should be it's own "Renew" button.